tests/test_admin.py

```diff
@@ -0,0 +1,98 @@
+from pathlib import Path
+
+from django.contrib.admin import AdminSite
+from django.contrib.auth.models import User
+from django.core.files import File
+from django.test import RequestFactory, TestCase
+from django.utils.translation import gettext_lazy as _
+
+from privates.test import temp_private_root
+from pyquery import PyQuery as pq
+
+from simple_certmanager.admin import CertificateAdmin
+from simple_certmanager.constants import CertificateTypes
+from simple_certmanager.models import Certificate
+
+TEST_FILES = Path(__file__).parent / "data"
+
+
+@temp_private_root()
+class AdminTests(TestCase):
```

Comment:
for the next time - we prefer writing tests in pytest style for libraries, pytest-django provides many useful tools to make tests easier to read and maintain.